### PR TITLE
pkg/proxy: add sig-network-approvers/sig-network-reviewers to OWNERS files

### PR DESCRIPTION
**Pull request by dcbw**
Use sig-network-[approvers | reviewers] in most of the proxy OWNERS files. Leave pkg/proxy/api with Tim though.

/kind cleanup
```release-note
NONE
```
@thockin

### CODE REVIEW

**Comment by dcbw** [comment]
@thockin what do you think?

File: pkg/proxy/OWNERS [hunk 1]
```diff
@@ -1,18 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- thockin
+- sig-network-approvers
 - matchstick
```
Comment (thockin) [inline pkg/proxy/OWNERS]: probably OK to remove this one

File: pkg/proxy/OWNERS [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 reviewers:
-- thockin
+- sig-network-reviewers
 - lavalamp
```
Comment (thockin) [inline pkg/proxy/OWNERS]: we should probably remove everyone here and offer @m1093782566 a spot in reviewers.

Comment (m1093782566) [reply pkg/proxy/OWNERS]: Thanks Tim :)

Comment (dcbw) [reply pkg/proxy/OWNERS]: @thockin I added @m1093782566 to sig-network-reviewers; he was already in the list for pkg/proxy/OWNERS so I assume you meant to just add him to signet review.

File: pkg/proxy/OWNERS (hunk 1 above)
Comment (dcbw) [reply pkg/proxy/OWNERS]: done

**Comment by dcbw** [comment]
@thockin updated, PTAL thanks!

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**